src/openforms/registrations/contrib/objects_api/templatetags/registrations/contrib/objects_api/json_summary.py

```diff
@@ -0,0 +1,18 @@
+import json
+
+from django import template
+from django.utils.safestring import SafeString
+
+from openforms.formio.rendering.structured import render_json
+
+register = template.Library()
+
+
+@register.simple_tag(takes_context=True)
+def json_summary(context):
+    submission = context.get("_submission")
+    if not submission:
+        return {}
+
+    get_json_renderer = render_json(submission)
```

Comment:
please avoid variable names that look like function names

```suggestion
    json_data = render_json(submission)
```